templates/AssetInventory/CardView/CardView.tsx

```diff
@@ -69,6 +69,12 @@ export const CardView = ({ id, instance }: CarViewProps) => {
                 </HvCardContent>
               </div>
             </div>
+            <HvCardMedia
+              component="img"
+              alt="Compressor"
```

Comment:
The `alt` attribute is always "Compressor". I think we should get this value from `row.original` in order to have an `alt` value that changes with the image.